src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -431,7 +440,14 @@ class DatePickerInput extends React.PureComponent {
     }
 
     onInputKeyDown(event) {
-        if (event.keyCode === 9 || event.keyCode === 13) {
+        const {
+            hideCalendarPickerOnKeyDown,
+        } = this.props;
+
+        const shouldHideCalendarPicker = hideCalendarPickerOnKeyDown ||
+            (event.keyCode === 9 || event.keyCode === 13);
```

Comment:
NAB (esp. since it was existing code) -- would be nice to use constants for the key codes instead of numeric literals.

Reply:
Fixed: https://github.com/saddlebackdev/react-cm-ui/pull/458/commits/18bb844efde21bde7a0e9d6516c6013c42418ff8#diff-4d384323ae0810e29196d88f93fb17cd2cda4bdff55a0f241e7558f675d82cc2L448-R449